src/Backend/OnboardingInsuranceAPI/Areas/User/UserData.cs

```diff
@@ -0,0 +1,27 @@
+using System.Text.Json.Serialization;
+
+namespace OnboardingInsuranceAPI.Areas.User;
+
+public record UserData
+{
+    [JsonPropertyName("pid")]
+    public string Pid { get; init; }
+    [JsonPropertyName("name")]
+    public string? Name { get; init; }
+    [JsonPropertyName("surname")]
+    public string? Surname { get; init; }
+    [JsonPropertyName("birthdate")]
+    public string? Birthdate { get; init; }
```

Comment:
Since the Birthdate is a string you should probably do a check if it's a valid date when getting it in a request.

Reply:
Ok